From 80c42b2a3228f64a19d7b14ba7034cda751dab14 Mon Sep 17 00:00:00 2001
From: Tobias Schlemmer <keinstein@users.sourceforge.net>
Date: Wed, 24 Jan 2018 09:59:25 +0100
Subject: update CFLAGS

---
 win32/mingwin32.mak | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/win32/mingwin32.mak b/win32/mingwin32.mak
index b44d332..538c7f3 100644
--- a/win32/mingwin32.mak
+++ b/win32/mingwin32.mak
@@ -25,7 +25,7 @@ ifeq ($(DEBUG),Y)
 	CFLAGS  = -g -Wall -DPDCDEBUG
 	LDFLAGS = -g
 else
-	CFLAGS  = -O2 -Wall
+	CFLAGS = -O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions --param=ssp-buffer-size=4
 	LDFLAGS =
 endif
 
